### PR TITLE
FLEET-1107 Crowd funding seeder add below header

### PR DESCRIPTION
**Pull request by fleet-app**

### CODE REVIEW

File: src/components/home/homescreen.js [hunk 1]
```diff
@@ -183,7 +185,21 @@ export default class Homescreen extends Component {
     }
 
     return (
+      
       <div>
+       <div id="crowdfund" className="cfund">
+              <div className="col-xs-6 col-sm-offset-3 col-sm-6 text-center">
+                  <h2 className="cfund_text_main secondary-text-color"> { LocalizationService.formatMessage('crowdfunding.ownfleet') } 🚀</h2>
```
Comment (plzen) [inline src/components/home/homescreen.js]: I think 🚀 should be a part of localized message

File: src/components/home/homescreen.js [hunk 2]
```diff
@@ -183,7 +185,21 @@ export default class Homescreen extends Component {
...
+                  <h4 className="cfund_text_secondary">{ LocalizationService.formatMessage('crowdfunding.seedrs') }&nbsp;
```
Comment (plzen) [inline src/components/home/homescreen.js]: Why can't you use single localized message here?

File: src/locales/locales.json [hunk 3]
```diff
@@ -106,6 +106,11 @@
     "header.sign_up": "Sign up",
     "header.switch_to_role": "Switch to {role}",
 
+    "crowdfunding.ownfleet":"Own a piece of fleet!",
```
Comment (plzen) [inline src/locales/locales.json]: Please, follow code style from other localized strings, like:
`"crowdfunding.ownfleet": "Own a piece of fleet!",` - space between semicolon and localization message

File: src/components/home/homescreen.js (hunk 2 above)
Comment (fleet-app) [reply src/components/home/homescreen.js]: I am follow on the UX design

image

File: src/locales/locales.json (hunk 3 above)
Comment (fleet-app) [reply src/locales/locales.json]: Done.

File: src/components/home/homescreen.js (hunk 1 above)
Comment (fleet-app) [reply src/components/home/homescreen.js]: Done

File: src/components/home/homescreen.js (hunk 2 above)
Comment (plzen) [reply src/components/home/homescreen.js]: Looks like you can use translation with `We're crowdfunding on Seedrs. Invest to join us on our mission to transform car hire.`

Comment (plzen) [reply src/components/home/homescreen.js]: Why did you split it?

Comment (fleet-app) [reply src/components/home/homescreen.js]: Fix done

**Pull request merged**